dace/transformation/transformation.py

```diff
@@ -386,20 +525,20 @@ class SubgraphTransformation(object):
                            desc='Subgraph in transformation instance')
 
     def __init__(self,
-                 subgraph: Union[Set[int], SubgraphView],
+                 subgraph: Union[Set[int], gr.SubgraphView],
                  sdfg_id: int = None,
                  state_id: int = None):
-        if (not isinstance(subgraph, (SubgraphView, SDFG, SDFGState))
+        if (not isinstance(subgraph, (gr.SubgraphView, SDFG, SDFGState))
```

Comment:
SDFG and SDFGState without module name

Reply:
Same as above, I won't be able to fix all existing cases here.